src/Runner.Worker/JobExtension.cs

```diff
@@ -316,6 +316,26 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                         }
                     }
 
+                    if (message.Variables.TryGetValue("system.workflowFileFullPath", out VariableValue workflowFileFullPath))
+                    {
+                        context.Output($"uses: {workflowFileFullPath.Value}");
+                        if (message.ContextData.TryGetValue("inputs", out var pipelineContextData)) {
+                            var inputs = pipelineContextData.AssertDictionary("inputs");
+                            if (inputs.Any()) {
+                                context.Output($"##[group] inputs");
+                                context.Output("with:");
```

Comment:
```suggestion
```
in case of workflow dispatch we have inputs from dispatch inputs too.

So i think its better we just mention inputs and remove `with`, so that it wont be confusing for customers ?